src/Components/EssentialPage.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import { View, StyleSheet, Text, TouchableOpacity, FlatList, ImageBackground } from 'react-native';
-import { Picker } from '@react-native-community/picker';
+import { Picker } from '@react-native-picker/picker';
```

Comment:
There is this error while running the tests

```
    SyntaxError: Unexpected token 'export'

    > 3 | import { Picker } from '@react-native-picker/picker';
        | ^

```

Could be related to https://github.com/react-native-picker/picker/issues/98

Reply:
Came from transformIgnorePattern not being defined: https://docs.expo.io/guides/testing-with-jest/#jest-configuration